decidim/decidim
# Fix cache when changing background image on homepage's Hero's cell

#### :tophat: What? Why?

We're being too aggressive caching on the Homepage's Hero Image Content block. This PR fixes that. 


#### :pushpin: Related Issues

- Fixes #9283

#### Testing

1. Enable the cache with  `bin/rails dev:cache` 
2. Sign in as admin 
3. Go to http://localhost:3000/admin/organization/homepage/content_blocks/hero/edit
4. Upload a background image
5. Click on the "Update" button
6. See that the image in this homepage block isn't updated

:hearts: Thank you!
